config/types.go

```diff
@@ -1,10 +1,11 @@
 package config
 
 import (
+	"runtime"
```

Comment:
It was already there, just got moved around a little. Looks like like it's used here https://github.com/joshwget/os/blob/bb25737ade67ae25cd4391460aadec179df80bbd/config/types.go#L64